static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-liveblog.scss

```diff
@@ -80,5 +80,5 @@
 
 @include overide-live-blog-colours(news, $live-garnett-main-1, $live-garnett-main-2);
 @include overide-live-blog-colours(sport, $live-garnett-sport1, $live-garnett-sport2);
-@include overide-live-blog-colours(arts, $live-garnett-arts1, $live-garnett-arts2);
+@include overide-live-blog-colours(culture, $live-garnett-culture1, $live-garnett-culture2);
```

Comment:
@zef - we need to pass "arts" to the mixin `overide-live-blog-colours` instead of "culture".